R/readpolicy_general.R

```diff
@@ -0,0 +1,46 @@
+#' readpolicy_general
+#'
+#' @param initial belief to compute dot product with alpha vectors.
+#' @param fs the initial fully observable state
+#' @param Num_fs dimension of the fully observable state
+#' @param file path to file to read in policy
+#' @return a list of the max alpha and associated alpha action, as well as all alpha vectors and associated actions.
+#' @importFrom xml2 read_xml xml_find_all xml_contents xml_attr
+#' @export
```

Comment:
Ideally the documentation should also include an example use with `@examples`, provided it runs quickly (< 5s).  These are also run automatically by the check.